alphadia/workflow/manager.py

```diff
@@ -750,3 +752,43 @@ def fit(self, update_dict):
 def column_hash(columns):
     columns.sort()
     return xxhash.xxh64_hexdigest("".join(columns))
+
+
+class TimingManager(BaseManager):
+    def __init__(
+        self,
+        path: None | str = None,
+        load_from_file: bool = True,
+        **kwargs,
+    ):
+        super().__init__(path=path, load_from_file=load_from_file, **kwargs)
+        self.reporter.log_string(f"Initializing {self.__class__.__name__}")
+        self.reporter.log_event("initializing", {"name": f"{self.__class__.__name__}"})
+
+        if not self.is_loaded_from_file:
+            self.__dict__.update(
```

Comment:
I know the other managers use the `__dict__` idiom, but could we start trying to avoid it?
(unless there's a good reason, @GeorgWa ?)

cf. https://stackoverflow.com/a/60105123

Reply:
from that I see, this could be a dictionary stored in an instance variable